File: Praktikum C/PRAK110-2210817210033-HariOctavianDelrossi.c
#include <stdio.h>
int main()
{
    float alas, tinggi, A, B, C, kel, L;
    alas=5;
    tinggi=12;
    A=alas;
    B=sqrt(alas*alas+tinggi*tinggi);
    C=tinggi;
    kel=A+B+C;
    L=A*C/2;
    printf("Diketahui :\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %.0f cm\n", A);
    printf("Sisi B = %.0f cm\n", B);
    printf("Sisi C = %.0f cm\n", C);
    printf("Keliling = %.0f cm\n", kel);
    printf("Luas = %.0f cm", L);
    return 0;
}

File: Praktikum Python/PRAK110-2210817210033-HariOctavianDelrossi.py
import math
alas=5
tinggi=12
A=alas
B=math.sqrt(alas*alas+tinggi*tinggi)
C=tinggi
kel=A+B+C
L=A*C/2
print('Diketahui :')
print('Alas =', alas, 'cm')
print('Tinggi =', tinggi, 'cm\n')
print('Jawab :')
print('Sisi A =', A, 'cm')
print('Sisi B = ', end='')
print('%.0f' %B, 'cm')
print('Sisi C =', C, 'cm')
print('Keliling = ', end='')
print('%.0f' %kel, 'cm')
print('Luas = ', end='')
print('%.0f' %L, 'cm')